api/src/integrations/core/mine-utils.js

```diff
@@ -58,6 +59,8 @@ class Mines extends BaseRecordUtils {
       location : { type: 'Point', coordinates: mineRecord.coordinates },
       permittee: this.getParty(MINE_PARTY_PERMITTEE, mineRecord, permit),
       type: '',
+      // TODO:  Without this, subsequent runs of the mine importer will duplicate mines every time it's run
```

Comment:
Does this still need to be done?

Reply:
Yes

Reply:
Ticket: https://bcmines.atlassian.net/browse/NRPT-232